### PR TITLE
Atlassian cloud it admin

### PR DESCRIPTION
**Pull request by sonaaru**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A. New Integration.

## Description
Initial commit.

## Screenshots
N/A

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Review by dantavori — CHANGES_REQUESTED** [review]
see my notes - also please generate the readme file of the integration

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+            proxy=proxy,
+            headers=headers)
+
+        if command in commands:
+            human_readable, outputs, raw_response = commands[command](client, demisto.args())
+            return_outputs(readable_output=human_readable, outputs=outputs, raw_response=raw_response)
+
+    # Log exceptions
+    except Exception:
+        demisto.error(f'Failed to execute {demisto.command()} command. Traceback: {traceback.format_exc()}')
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py:610]: > done

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+
+def disable_user_command(client, args):
+    return enablee_disable_user_command(client, args, False)
+
+
+def enable_user_command(client, args):
+    return enablee_disable_user_command(client, args, True)
+
+
+def enablee_disable_user_command(client, args, is_active):
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py:505]: done

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py [hunk 3]
```diff
@@ -0,0 +1,615 @@
...
+        generic_iam_context.data
+    )
+
+
+def disable_user_command(client, args):
+    return enablee_disable_user_command(client, args, False)
+
+
+def enable_user_command(client, args):
+    return enablee_disable_user_command(client, args, True)
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py:502]: done

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py [hunk 4]
```diff
@@ -0,0 +1,615 @@
...
+
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def disable_user_command(client, args):
+    return enablee_disable_user_command(client, args, False)
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py:498]: done

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py [hunk 5]
```diff
@@ -0,0 +1,615 @@
...
+    """result = client.http_request('DBot')
+    if res.status_code == 200:
+        return 'ok'
+    else:
+        raise Exception"""
+
+    uri = f'/scim/directory/{encode_string_results(client.directory_id)}/Users/1234'
+    res = client.http_request(method='GET', url_suffix=uri)
+    if res.status_code == 403:
+        err_msg = 'Invalid url: ' + client.base_url + f'/scim/directory/{encode_string_results(client.directory_id)}'
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin.py:249]: done

File: Packs/AtlassianCloudITAdmin/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Atlassian Cloud IT Admin",
+    "description": "Atlassian Cloud IT Admin",
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/pack_metadata.json:3]: done

File: Packs/AtlassianCloudITAdmin/TestPlaybooks/playbook-Atlassian_Cloud_IT_Admin.yml [hunk 7]
```diff
@@ -0,0 +1,620 @@
+id: testplaybook-Atlassian_Cloud_IT_Admin
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/TestPlaybooks/playbook-Atlassian_Cloud_IT_Admin.yml:1]: No. Moving the playbook to be under NonCircleTests dir.

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/CHANGELOG.md]: done

File: Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin_description.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+•	From your organization at admin.atlassian.com, select Settings and then API keys.
+•	Click Create API key in the top right.
+•	Enter a name that you’ll remember to identify the API key.
+•	By default, the key expires one week from today. If you’d like to change the expiration date, pick a new date under Expires on. You’re unable to select a date longer than a year from the date of creation.
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Packs/AtlassianCloudITAdmin/Integrations/Atlassian_Cloud_IT_Admin/Atlassian_Cloud_IT_Admin_description.md:4]: https://confluence.atlassian.com/cloud/create-an-admin-api-key-969537932.html

File: Tests/conf.json [hunk 10]
```diff
@@ -2,7 +2,19 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
-	{
+        {
```
[the remark this answers is not in the document]
Comment (sonaaru) [reply Tests/conf.json:5]: We wrongly updated this file so reverted the changes

**Review by dantavori — APPROVED** [review]

**Pull request merged**